src/interfaces/LogInterface.ts

```diff
@@ -0,0 +1,6 @@
+export interface LogInterface {
```

Comment:
We should avoid using `Interface` in types/interface names. Given we don't have multiple implementations of loggers we could actually remove this entirely. `Logger` defines it's own interface and avoids the issue we have now were we have to update types in the implementation interface (`any` -> `unknown` comment)